internal/tiltfile/docker_compose.go

```diff
@@ -301,7 +301,7 @@ func (s *tiltfileState) dcServiceToManifest(service *dcService, dcSet dcResource
 	paths = append(paths, dcInfo.LocalPaths()...)
 	paths = append(paths, filepath.Dir(dcSet.tiltfilePath))
 
-	dcInfo = dcInfo.WithDockerignores(s.dockerignoresFromPathsAndContextFilters(paths, []string{}, []string{}))
```

Comment:
i think you need to pass `service.DfPath` here

Reply:
That's exactly what I've been trying to figure out for the last half hour. I wish I'd seen your comment sooner!